src/CftfBundle/Resources/views/DocTree/export.json.twig

```diff
@@ -43,11 +43,20 @@
     {% if concepts is not empty -%}
     {#- TODO: CFConcepts -#}
     ,"CFConcepts": [
-    ]
     {%- endif %}
     {% if licences is not empty -%}
-    {#- TODO: CFLicenses -#}
     ,"CFLicenses": [
+    {% for licence in licences %}
+    {% if not loop.first %},{% endif %}
+        {
+            "identifier": {{ licence.identifier |json_encode |raw }}
+            , "uri": {{ licence.uri |json_encode |raw }}
+            , "title": {{ licence.title |json_encode |raw }}
+            , "description": {{ licence.description |json_encode |raw }}
+            , "licenceText": {{ licence.licenceText |json_encode |raw }}
```

Comment:
The name of the field in the JSON should be `licenseText` as defined in the spec.